src/testUtil/TestCase.ts

```diff
@@ -36,17 +36,19 @@ export type TestCaseFixture = {
   marksToCheck?: string[];
 
   initialState: TestCaseSnapshot;
-  finalState: TestCaseSnapshot;
+  finalState: TestCaseSnapshot | null;
   returnValue: unknown;
   /** Inferred full targets added for context; not currently used in testing */
   fullTargets: Target[];
+  errorReturned: boolean;
```

Comment:
I would capture name of error, and then check for that in test.  Can use `(err as Error).name`.  I think I'd have this field be `thrownError: ErrorInfo`, and to just have `name: string` field on the `ErrorInfo` type for now